qemu/tests/pci_bridge_block_stress.py

```diff
@@ -0,0 +1,66 @@
+import logging
+
+from autotest.client.shared import error
+from virttest import utils_test
+from virttest import env_process
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    [pci-bridge]Check stress when 31 block devices attached to 1 pci-bridge, this case will:
+    1) Create one pci-bridge in configuration file.
+    2) Create disks connected to that pci-bridge.
+    3) Start the guest.
+    4) Check 'info block'.
+    5) Read and write data on disks under pci bridge.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error.context("Modify params!", logging.info)
+    image_parent_bus = params.get("image_parent_bus")
+    image_num = int(params.get("image_num", 0))
+    if image_num != 0:
+        for index in xrange(image_num):
+            image = "stg%s" % index
+            params["images"] = ' '.join([params["images"], image])
+            params["disk_pci_bus_%s" % image] = image_parent_bus
+            params["image_name_%s" % image] = "images/%s" % image
+            params["image_size_%s" % image] = "100M"
+            params["force_create_image_%s" % image] = "yes"
+            params["remove_image_%s" % image] = "yes"
+            params["blk_extra_params_%s" % image] = "serial=TARGET_DISK%s" % index
+
+    env_process.process_images(env_process.preprocess_image, test, params)
+    env_process.preprocess_vm(test, params, env, params["main_vm"])
+
+    error.context("Get the main VM!", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    login_timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=login_timeout)
+
+    error.context("Check 'info block'!", logging.info)
+    monitor_info_block = vm.monitor.info_block(False)
+    if image_num + 1 != len(monitor_info_block.keys()):
+        raise error.TestFail("Check 'info block' failed!")
+    error.context("Check 'info block' succeed!", logging.info)
```

Comment:
here you can use logging.info instead of error.context
and suggest to use error_context.context to instead of error.context